securedrop_client/gui/widgets.py

```diff
@@ -837,7 +839,7 @@ def update(self):
         if self.source.document_count == 0:
             self.attached.hide()
 
-    def delete_source(self, event):
+    def delete_source(self, event: Any) -> None:
```

Comment:
This can be `QEvent` (I think). We might be able to scope it tighter if we check what `delete_source` is attached to.